### ISSUE TITLE
There is also another lr_scheduler inside the train_one_epoch function ? any reason? Thanks

### ISSUE THREAD

**Issue by manoja328** [pytorch/vision#1213]
https://github.com/pytorch/vision/blob/64ccfe34044e020b2120a4fcfa2651bd8d802a40/references/detection/train.py#L117

**Comment by EmilioOldenziel** [comment]
@manoja328 its only used in the first epoch to warmup the lr from 0 to the set lr. Check lines 19 to 24 in engine.py on how it is initialized.

**Comment by fmassa** [comment]
@EmilioOldenziel is right, this is for warmup.

I also considered if we should follow a similar approach as in the `video_classification` training script, where we pass a combined `lr_scheduler` instead, so that we don't need to have two schedulers, but there are no significant gains from now, but requires having to write a new scheduler.
